src/main/java/com/openlattice/chronicle/ChronicleStudyApi.java

```diff
@@ -135,13 +185,16 @@ Void deleteStudyAndAllNeighbors(
     /**
      * Returns a file download containing preprocessed data.
      *
+     * @param organizationId         - Id of the organization to which study belongs
      * @param studyId                - the study id
      * @param participantEntityKeyId - the participant entity key id
      * @param fileType               - the type of file (csv, json) to return as the download
      * @return All participant data
      */
-    @GET( BASE + AUTHENTICATED + PARTICIPANT_PATH + DATA_PATH + STUDY_ID_PATH + ENTITY_KEY_ID_PATH + PREPROCESSED_PATH )
+    @GET( BASE + AUTHENTICATED + PARTICIPANT_PATH + DATA_PATH + ORGANIZATION_ID_PATH + STUDY_ID_PATH
```

Comment:
I think you'll need to keep `PREPROCESSED_PATH` as part of the path here, otherwise the controller won't be able to differentiate between an incoming request here vs to `getAllParticipantData` above